### PR TITLE
874252-Volume_1_features

### PR DESCRIPTION
**Pull request by SivaranjithN**
Added UG documents for below features in mentioned controls.
### Autocomplete
- ShowBorder
- HorizontalTextAlignment
- VerticaltextAlignment

### ComboBox
- ShowBorder
- HorizontalTextAlignment
- VerticalTextAlignment

### MaskedEntry
- Stroke
- ShowBorder
- HorizontalTextAlignment
- VerticalTextAlignment

### NumericEntry
- ShowBorder
- HorizontalTextAlignment
- VerticalTextAlignment

### Button
- RTL support

### TextInputLayout
- PasswordVisibilityToggled event

### Chip
- IsSelected

### CODE REVIEW

File: MAUI/Autocomplete/UI-Customization.md [hunk 1]
```diff
@@ -859,3 +859,58 @@ autocomplete.CursorPosition = 4;
 {% endtabs %}
 
 N> Two way binding not supported in Android platform.
+
+## ShowBorder
+
+The `ShowBorder` property is used to modify the visibility of border. The following code example demonstrates how to change the border visibility in [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). The default value of the `ShowBorder` property is **true**.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/UI-Customization.md:865]: please modify the content,

The `ShowBorder` property is used to modify the visibility of the border and its default value is **true**. The following code example demonstrates how to change the border visibility in [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html).

File: MAUI/Autocomplete/UI-Customization.md [hunk 2]
```diff
@@ -859,3 +859,58 @@ autocomplete.CursorPosition = 4;
 {% endtabs %}
 
 N> Two way binding not supported in Android platform.
+
+## ShowBorder
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/UI-Customization.md:863]: Change title to border visibility

File: MAUI/Autocomplete/UI-Customization.md [hunk 3]
```diff
@@ -859,3 +859,58 @@ autocomplete.CursorPosition = 4;
...
+
+The `HorizontalTextAlignment` and `VerticalTextAlignment` properties are used to customize the alignment of text in [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). This property accepts values from the TextAlignment enumeration, which includes options like `Start`, `Center`, and `End`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autoComplete"
+                        WidthRequest="250" 
+                        HeightRequest="50"
+                        HorizontalTextAlignment="End"
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/UI-Customization.md:899]: Set horizontal text alignment to the center and modify image

File: MAUI/Autocomplete/UI-Customization.md [hunk 4]
```diff
@@ -859,3 +859,58 @@ autocomplete.CursorPosition = 4;
...
+autocomplete.ShowBorder = false;
+
+{% endhighlight %}
+{% endtabs %}
+
+![ShowBorder](Images/UICustomization/ShowBorder.png)
+
+### TextAlignment
+
+The `HorizontalTextAlignment` and `VerticalTextAlignment` properties are used to customize the alignment of text in [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). This property accepts values from the TextAlignment enumeration, which includes options like `Start`, `Center`, and `End`.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/UI-Customization.md:891]: please modify the sentence structure like below

The auto-complete provides support to customize the text alignment by using the horizontal text alignment and vertical text alignment properties.

File: MAUI/Button/Customization.md [hunk 5]
```diff
@@ -669,3 +669,37 @@ public class CommandDemoViewModel : INotifyPropertyChanged
 
 {% endhighlight %}
 {% endtabs %}
+
+## RTL support
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Button/Customization.md:673]: Change title to flow direction

File: MAUI/Button/Customization.md [hunk 6]
```diff
@@ -669,3 +669,37 @@ public class CommandDemoViewModel : INotifyPropertyChanged
...
+
+{% highlight c# %}
+SfButton button = new SfButton()
+{
+    Text = "Add to cart",
+    TextColor = Colors.Black,
+    HorizontalTextAlignment = LayoutOptions.Start,
+    ImageSource = "add_to_card.png",
+    ShowIcon = true,
+    ImageAlignment = Alignment.Left
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Button/Customization.md:698]: Add flow direction

File: MAUI/Chips/Customization.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 layout: post
 title: Customization in .NET MAUI Chips control | Syncfusion
 description: Learn about Customization support in Syncfusion Essential Studio .NET MAUI Chips control, its elements and more.
-platform: .NET MAUI
+platform: maui
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:5]: why do you change it

File: MAUI/Chips/Customization.md [hunk 8]
```diff
@@ -1806,4 +1806,8 @@ using Syncfusion.Maui.Core;
 
 N> The default value of ChipImageSize is [`18d`].
 
+## IsSelected
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:1809]: Add this title under the selection properties

File: MAUI/Masked-Entry/Basic-Features.md [hunk 9]
```diff
@@ -200,5 +200,88 @@ maskedEntry.TextColor =  Colors.Green;
 
 ![TextColor](MaskedEntry_Images/MaskedEntry_TextColor.png)
 
+## Setting Stroke
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Masked-Entry/Basic-Features.md:203]: check the title

File: MAUI/Masked-Entry/Basic-Features.md [hunk 10]
```diff
@@ -200,5 +200,88 @@ maskedEntry.TextColor =  Colors.Green;
...
+
+The `Stroke` property is used to modify the visibility of border. The following code example demonstrates how to change the border visibility in [`SfMaskedEntry`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfMaskedEntry.html).
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Masked-Entry/Basic-Features.md:205]: please check the description of this title.

File: MAUI/TextInputLayout/Getting-Started.md [hunk 11]
```diff
@@ -162,3 +162,39 @@ N> Password visibility toggle can be enabled only for [Entry](https://learn.micr
 You can find the complete getting started sample from this [link.](https://github.com/SyncfusionExamples/maui-textinputlayout-samples)
 
 N> You can refer to our [.NET MAUI Text Input Layout](https://www.syncfusion.com/maui-controls/maui-textinputlayout) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Text Input Layout Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/TextInputLayout) that shows you how to render the Text Input Layout in .NET MAUI.
+
+## PasswordVisibilityToggled Event
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TextInputLayout/Getting-Started.md:166]: Move this section under Events folder

File: MAUI/Chips/Customization.md [hunk 12]
```diff
@@ -1806,4 +1806,8 @@ using Syncfusion.Maui.Core;
...
 
+The `IsSelected` property is a Boolean public property available in the [`SfChip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfChip.html). This property indicates whether a chip has been selected in choice and filter types.
+
+The IsSelected property serves as a marker for the selected state of a chip within choice and filter types. When set to **true**, it indicates that the chip has been selected, and when set to **false**, it indicates that the chip is not selected.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:1813]: We unable to set the IsSelected value, so remove the unnecessary content.

File: MAUI/Chips/Customization.md (hunk 7 above)
Comment (SivaranjithN) [reply MAUI/Chips/Customization.md:5]: Addressed the comments

File: MAUI/Chips/Customization.md (hunk 8 above)
Comment (SivaranjithN) [reply MAUI/Chips/Customization.md:1809]: Addressed the comments

File: MAUI/Chips/Customization.md (hunk 12 above)
Comment (SivaranjithN) [reply MAUI/Chips/Customization.md]: Addressed the comments.

File: MAUI/TextInputLayout/Getting-Started.md (hunk 11 above)
Comment (SivaranjithN) [reply MAUI/TextInputLayout/Getting-Started.md]: Addressed the comments.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 10 above)
Comment (SivaranjithN) [reply MAUI/Masked-Entry/Basic-Features.md]: Addressed the comments.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 9 above)
Comment (SivaranjithN) [reply MAUI/Masked-Entry/Basic-Features.md]: Addressed the comments.

File: MAUI/Button/Customization.md (hunk 6 above)
Comment (SivaranjithN) [reply MAUI/Button/Customization.md]: Addressed the comments.

File: MAUI/Autocomplete/UI-Customization.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/UI-Customization.md]: Addressed the comments.

File: MAUI/Autocomplete/UI-Customization.md (hunk 2 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/UI-Customization.md]: Addressed the comments.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**